### PR TITLE
feat: support `fnm install --latest`

### PR DESCRIPTION
**Pull request by nzhl**
This should help to support install the latest node version using `fnm install --latest`
Just as  https://github.com/Schniz/fnm/issues/797 expected.

### CODE REVIEW

File: src/commands/install.rs [hunk 1]
```diff
@@ -153,6 +175,8 @@ pub enum Error {
     CantFindNodeVersion { requested_version: UserVersion },
     #[error("Can't find relevant LTS named {}", lts_type)]
     CantFindRelevantLts { lts_type: crate::lts::LtsType },
+    #[error("Can't find latest version")]
+    CantFindLatest,
```
Comment (Schniz) [inline src/commands/install.rs:179]: I wonder if we should be more explicit about the issue, like: "can't find any versions in the upstream version index"

**Review by Schniz — COMMENTED** [review]
Can you please run `pnpm generate-command-docs` to update the `commands.md` file?

I also had a small suggestion for the error message.

other than that, <picture data-single-emoji=":lgtm:" title=":lgtm:">:lgtm:</picture>

I wonder if we can add a test for it somehow. Given that the version is dynamic it might be problematic, but maybe just a smoke test with no expectations other than "not fail"?

**Comment by Schniz** [comment]
Also, thank you very much for stepping in and contributing! <picture data-single-emoji=":clap_sunglasses_+1:" title=":clap_sunglasses_+1:">:clap_sunglasses_+1:</picture> 

can you run `cargo fmt` and fix `cargo clippy` issues? <picture data-single-emoji=":pray_parrot:" title=":pray_parrot:">:pray_parrot:</picture>

File: src/commands/install.rs (hunk 1 above)
Comment (nzhl) [reply src/commands/install.rs:179]: resolved, thx for suggestion

**Comment by nzhl** [comment]
> Also, thank you very much for stepping in and contributing! :clap_sunglasses_+1:
> 
> can you run `cargo fmt` and fix `cargo clippy` issues? :pray_parrot:


Thx for suggestion,  i have added smoke test and run `cargo fmt` and `cargo clippy`.

**Comment by Schniz** [comment]
The commands file is not up to date because of the version change :( can you rerun the script?

**Comment by nzhl** [comment]
> change

sure

**Comment by nzhl** [comment]
> The commands file is not up to date because of the version change :( can you rerun the script?

done

**Comment by nzhl** [comment]
> The commands file is not up to date because of the version change :( can you rerun the script?

sorry, one more question. Do i need to bump version manually ?

File: docs/commands.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # `fnm`
 
 ```
-fnm 1.32.0
+fnm 1.31.1
```
Comment (Schniz) [inline docs/commands.md:4]: I don’t understand why this is not updated, if you’re rebased from main and ran the script. Maybe something is funky there and I’ll fix it?

Comment (nzhl) [reply docs/commands.md]: I think it's updated rn after i run it one more time.

**Comment by Schniz** [comment]
Amazing! Thank you so much.

**Pull request merged**